backend/routes_bulk_import.py

```
@@ -0,0 +1,347 @@
+"""
+from flask import request, jsonify, make_response
+from functools import wraps
+from utils.bulk_import import (
+    bulk_import_tools, 
+    bulk_import_chemicals, 
+    generate_tool_template, 
+    generate_chemical_template,
+    BulkImportError
+)
+from utils.validation import ValidationError
+from auth import JWTManager, admin_required as jwt_admin_required
```

Ruff (0.11.9)

`auth.admin_required` imported but unused

Remove unused import: `auth.admin_required`

(F401)